test/integration/authorization-test.ts

```diff
@@ -27,6 +27,30 @@ describe('authorization headers', function() {
     });
   });
 
+  describe('when header is set in a custom generateAuthHeader', function() {
+    let originalHeaderFn = Person.generateAuthHeader;
+    beforeEach(function() {
+      ApplicationRecord.jwt = 'cu570m70k3n';
+      Person.generateAuthHeader = function(token) {
+        return `Bearer ${token}`;
+      };
+    });
+
+    afterEach(function() {
+      fetchMock.restore();
+      Person.generateAuthHeader = originalHeaderFn;
+    });
+
+    it('sends the custom Authorization token in the request¡\'s headers', function(done) {
```

Comment:
So tiny, but could you make the outer quotes `"` so the inner quote can be `'` without escaping? After that we are good to go!